test/stress/n_workers_reputers_stress_test.go

```diff
@@ -0,0 +1,314 @@
+	} else {
+		m.T.Log(topicLog(topicId, "Funded", len(reputers), "reputer accounts."))
+	}
+
+	//  4. Wait until the topic has had minWaitingNumberofEpochs before starting to provide inferences for it
+	topic, err := getNonZeroTopicEpochLastRan(
+		m,
+		topicId,
+		5,
+		approximateSecondsBlockTime,
+	)
+	if err != nil {
+		m.T.Log(topicLog(topicId, "--- Failed getting a topic that was ran ---"))
+		require.NoError(m.T, err)
+	}
+
+	// Translate the epoch length into time
+	iterationTime := time.Duration(topic.EpochLength) * approximateSecondsBlockTime * iterationsInABatch
+
+	countWorkers := 0
+	countReputers := 0
+	//  5. Begin "iterations" inside the topic
+	for i := 0; i < maxIterations; i++ {
+		// 6. Fund the topic to give it money to make inferences
+		err := fundTopic(m, topicId, funder, topicFunds)
+		if err != nil {
+			m.T.Log(topicLog(topicId, "Funding topic failed: ", err))
+			if makeReport {
+				saveTopicError(topicId, err)
+			}
+		}
+		startIteration := time.Now()
+
+		m.T.Log(topicLog(topicId, "iteration: ", i, " / ", maxIterations))
+
+		// 7. Register the newly created accounts for this iteration
+		countWorkers = registerWorkersForIteration(
+			m,
+			topicId,
+			i,
+			workersPerIteration,
+			countWorkers,
+			maxWorkersPerTopic,
+			workers,
+			makeReport,
+		)
+		countReputers = registerReputersForIteration(
+			m,
+			topicId,
+			i,
+			reputersPerIteration,
+			countReputers,
+			maxReputersPerTopic,
+			reputers,
+			makeReport,
+		)
+
+		//  8. Generate and insert a worker bundle (adjust nonces if failure)
+		err = generateInsertWorkerBundle(
+			m,
+			topic,
+			workers,
+			retryBundleUploadTimes,
+			makeReport,
+		)
+		if err != nil {
+			m.T.Log(topicLog(topicId, "Error generate/inserting worker bundle: ", err))
+			if makeReport {
+				saveTopicError(topicId, err)
+			}
+		}
+
+		//  9. Generate and insert reputer bundle scoring workers
+		err = generateInsertReputerBulk(
+			m,
+			topic,
+			reputers,
+			workers,
+			retryBundleUploadTimes,
+			makeReport,
+		)
+		if err != nil {
+			m.T.Log(topicLog(topicId, "Error generate/inserting reputer bundle: ", err))
+			if makeReport {
+				saveTopicError(topicId, err)
+			}
+		}
+
+		//  10. Sleep for an epoch
+		elapsedIteration := time.Since(startIteration)
+		sleepingTime := iterationTime - elapsedIteration
+		m.T.Log(topicLog(topicId, time.Now(), " Sleeping...", sleepingTime, ", elapsed: ", elapsedIteration, " epoch length seconds: ", iterationTime))
+		time.Sleep(sleepingTime)
+	}
+
+	// 11. Check that the workers have been paid rewards
+	rewardedWorkersCount, err := checkWorkersReceivedRewards(
+		m,
+		topicId,
+		workers,
+		countWorkers,
+		maxIterations,
+		makeReport,
+	)
+	if err != nil {
+		m.T.Log(topicLog(topicId, "Error checking worker rewards: ", err))
+		if makeReport {
+			saveTopicError(topicId, err)
+		}
+	}
+
+	// 11. Check that the reputer have been paid rewards via a stake greater than the initial amount
```

Comment:
you have two 11's. Maybe just drop the numbers so we don't need to worry about changing the numbers every time we update the logic